pkg/printer/printer_test.go

```diff
@@ -373,29 +373,32 @@ func Test_getTimestamp(t *testing.T) {
 		{
 			name: "valid",
 			args: args{
-				f: &pb.Flow{
-					Time: &timestamp.Timestamp{
-						Seconds: 0,
-						Nanos:   0,
-					},
+				t: &timestamp.Timestamp{
+					Seconds: 0,
+					Nanos:   0,
 				},
 			},
 			want: "Jan  1 00:00:00.000",
 		},
 		{
 			name: "invalid",
-			args: args{},
+			args: args{
+				t: &timestamp.Timestamp{
+					Seconds: 0,
+					Nanos:   2_000_000_000,
```

Comment:
casting `X * time.Seconds` would probably be even more legible here

Reply:
It's supposed to be an invalid timestamp. So casting doesn't work. I used negative numbers instead now, to make it more legible that it is supposed to be a garbage value.